src/deploy/constants.ts

```diff
@@ -0,0 +1,8 @@
+
+export const ProxyKinds = {
```

Comment:
This is just an object with three mutually exclusive properties. Make this a struct instead

Reply:
what struct?
this is a map of constant variables to make sure we use the same strings everywhere. you can call it an NodeJS version of enum.